bin/generate.sh

```diff
@@ -0,0 +1,7 @@
+set -e
```

Comment:
We've recently [updated the spec](https://github.com/exercism/automated-analysis/blob/master/docs/docker.md#representers) to have this file be named `run.sh`.

Reply:
I don't know if I can see this somewhere in GitHub, but the executable bit also has to be set on this file.